DataManagementSystem/Agent/FTSAgent.py

```diff
@@ -664,6 +676,8 @@ def processRequest( self, request ):
 
     except escapeTry:
```

Comment:
To be pedantic, this exception class name should be rather EscapeTry or EscapeTryException